ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -420,6 +422,12 @@ def test_frontend_function(
     with_inplace
         if True, the function is only tested with direct inplace update back to
         the inputted array and ignore the value of with_out.
+    has_aliases
```

Comment:
This is not necessary, let's just use `all_aliases`. If it is `None` then we can assume that there are no aliases.